src/rasmus_svebestad_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Rasmus Svebestad'
+__email__ = 'rasmus.svebestad@nmbu.no'
+
+
+from random import randint
+import random
+
+
+class Walker:
+    def __init__(self, x0, home):
+        self.position = x0
+        self.home = home
+        self.steps = 0
+
+    def move(self):
+        self.position = self.position + randint(0, 1) * 2 - 1
+        self.steps = self.steps + 1
+
+    def is_at_home(self):
+        if self.position == self.home:
+            return True
+        return False
+
+    def get_position(self):
+        return self.position
+
+    def get_steps(self):
+        return self.steps
+
+
+class Simulation(Walker):
+    def __init__(self, start, home, seed):
+        self.position = start
+        self.home = home
+        self.seed = seed
+        self.steps = 0
+
+    def single_walk(self):
+        random.seed(self.seed)
```

Comment:
Put this in `__init__` as random seed should be set only once at the very first of everything.